audit/projects/k8s-staging-txtdirect/buckets/k8s-staging-txtdirect-gcb/iam.json

```diff
@@ -15,6 +15,12 @@
       ],
       "role": "roles/storage.legacyBucketReader"
     },
+    {
```

Comment:
This was, I think, the result of a bug that said reader when it meant writer.

fcd9c914

The readers should be removed.  Doing that now.